x-pack/legacy/plugins/security/public/views/management/users_grid/components/users_list_page.tsx

```diff
@@ -142,21 +142,21 @@ class UsersListPageUI extends Component<Props, State> {
         },
       },
       {
-        field: 'metadata._reserved',
+        field: 'metadata',
```

Comment:
Changed from `metadata._reserved` to `metadata` because EUI won't allow the field to sort on this potentially undefined field